frontend/src/app/components/wp-table/embedded/embedded-tables-macro.component.ts

```diff
@@ -29,10 +29,10 @@
 import {Component, ElementRef} from "@angular/core";
 import {WorkPackageTableConfigurationObject} from "core-components/wp-table/wp-table-configuration";
 
-export const wpEmbeddedTableEntrySelector = 'macro.embedded-table';
+export const wpEmbeddedTableMacroSelector = 'macro.embedded-table';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/686d48c8f4d6cfd75e9fd3e65594028c0af6989a/frontend/src/app/components/wp-table/embedded/embedded-tables-macro.component.ts#L33"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>